installer/pkg/config/parser.go

```diff
@@ -29,6 +29,7 @@ func ParseConfig(data []byte) (*Cluster, error) {
 			return nil, err
 		}
 		cluster.PullSecret = string(data)
+		cluster.PullSecretPath = ""
```

Comment:
Is this supposed to be in here?

Reply:
> Is this supposed to be in here?

Yeah.  We used to just copy the input config over.  Now that I'm saving the updates config (because it might contain a new libvirt URI), I need to clear `PullSecretPath` (after reading the content into `PullSecret`) to avoid hitting [this][1] when we reload the config for the `install` step. 

[1]: https://github.com/openshift/installer/blob/5f59fecda47abdafa442c534c88df86ac598de9c/installer/pkg/config/parser.go#L23-L24